README.md

```diff
@@ -1,2 +1,232 @@
+```
+
+### Verify the install succeeded
+
+Once the installation is finished, run the following command to verify the installation was succesfull:
+```bash
+# For a project based install:
+vendor/bin/phpcs -i
+
+# For a global install:
+phpcs -i
+```
+
+The output should look like this:
+```
+The installed coding standards are MySource, PEAR, PSR1, PSR12, PSR2, Squiz, Zend, WP-QA-Basic, WP-QA-Strict,
+PHPCompatibility, PHPCompatibilityParagonieRandomCompat, PHPCompatibilityParagonieSodiumCompat, PHPCompatibilityWP,
+WordPress, WordPress-Core, WordPress-Docs, WordPress-Extra and WPThemeReview
+```
+
+
+## Using the installed rulesets
+
+After installation, you can use any of the above listed PHPCS rulesets, or a combination of them, for your projects.
+
+* `WordPress-Core` checks code based on the guidelines described in the [WordPress PHP Coding Standards Handbook](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/).
+* `WordPress-Docs` checks code documentation based on the guidelines described in the [WordPress PHP Documentation Standards Handbook](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/).
+* `WordPress-Extra` is `WordPress-Core` + extra checks for common best practices, both for code in general, as well as WordPress specific best practices.
+* `WordPress` is a combination of the above three rulesets.
+
+For plugins and themes, using either `WordPress` or `WordPress-Extra` is recommended.
+
+* [`PHPCompatibilityWP`](https://github.com/PHPCompatibility/PHPCompatibilityWP) checks code for being PHP cross-version compatible while preventing false positives for PHP features polyfilled in WordPress itself.
+    This is the recommended PHPCompatibility ruleset to use for WordPress projects. The other PHPCompatibility rulesets listed are included in this ruleset.
+
+* [`WPThemeReview`](https://github.com/WPTRT/WPThemeReview) is specifically for WordPress themes and checks the code against the guidelines for submission to the Theme repository on wordpress.org as described in the [Theme Handbook](https://make.wordpress.org/themes/handbook/review/)
+
+It is strongly recommended to document the settings you use in a custom ruleset and to add some minimal sniff configuration for optimal results.
+
+An example ruleset which you can place in the root of your project, including documentation on what you should adjust, can be found in the [sample-project-ruleset](https://github.com/jrfnl/QA-WP-Projects/tree/master/sample-project-ruleset) directory.
+
+
+## Reviewing WordPress plugins and themes
+
+This repository comes with two native rulesets - `WP-QA-Basic` and `WP-QA-Strict` - which are specifically intended for reviewing WordPress plugins and themes without much knowledge of code.
+
+These rulesets do not look at the code style consistency of code. They will only evaluate whether code is well documented, tested and whether there are any code quality issues detected.
```

Comment:
This peaks my interest, where _do_ I find the code style sniffs?
Maybe add a hint about those.

Reply:
In `WordPress-Core` and `WordPress-Extra` which are listed in the [Using the installed rulesets](https://github.com/jrfnl/QA-WP-Projects/blob/9790cc76dfd5301e938188b26ea579dce3097b22/README.md#using-the-installed-rulesets) section above.

I'll add a mention there that those include code style checks.

Reply:
In `WordPress-Core` and `WordPress-Extra` which are listed in the [Using the installed rulesets](https://github.com/jrfnl/QA-WP-Projects/blob/9790cc76dfd5301e938188b26ea579dce3097b22/README.md#using-the-installed-rulesets) section above.

I'll add a mention there that those include code style checks.